api/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+class Api {
+    async getRequest(path) {
+        try {
+            return await axios.get(path);
+        } catch (e) {
+            return e.response;
```

Comment:
if exception was thrown response is returned. but most likely in this case the response will not be valid and you will not be able to retrieve name, weight, speed, etc.